### PR TITLE
Structure and routes

### PR DESCRIPTION
**Pull request by sagieramos**
### Key Features 
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the [API](https://api-football-standings.azharimm.dev/) is stored in the Redux store.
- Each page has a unique route

## [Live Demo](https://soccerhub.onrender.com/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +39,14 @@
...
 ### Key Features <a name="key-features"></a>
-- [Live Demo Link](https://space-2cht.onrender.com)
+
+- The project is a single-page application (SPA) built with React and Redux.
+- The data retrieved from the [API](https://api-football-standings.azharimm.dev/) is stored in the Redux store.
+- Each page has a unique route
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://soccerhub.onrender.com/)
```
Comment (DeliceLydia) [inline README.md:50]: - [ ] Please add in the README a recorded video presenting
- The features of the project
- A piece of the code you find interesting or proud of making

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @sagieramos, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo](https://soccerhub.onrender.com/)
+- [Live Demo](https://www.loom.com/share/95292335049d41549cf61d8cb9a2b95a?sid=4af47703-6bd8-4ea2-935a-12b355878f7b)
```
Comment (Ibizugbe) [inline README.md]: Nice job so far, your video presentation looks good👍🏼. In a bit to implement the corrections from the previous reviewer, I noticed you replaced the Live demo link with the video presentation. Kindly note that it is also a requirement that your readme file has a live demo link (A link to your deployed project) just as it should have a video presentation link. Kindly Add the link to your deployed project in your live demo link. Add a section called video presentation and add the link to your video presentation.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @sagieramos 👋🏼👋🏼👋🏼, 

require_changes


Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights✅✅✅

- [x] Video presentation looks good👍🏼.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [OPTIONAL] I suggest you add the install and deployment section as in the microverse readme template as this makes your readme more descriptive and professional😉.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @sagieramos, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. I'm pleased to announce that you've addressed all the concerns. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 

### Video Presentation and Live Demo Link: 🎥🔗
- Fantastic job ensuring that both the "Live Demo" and the "Video Presentation" links are present in your README. The clear distinction between the two links provides clarity for anyone viewing your project repository. Well done! 👏

### Overall: 🏆
Your attentiveness to feedback and swiftness in making the necessary adjustments is truly commendable. It's evident that you're dedicated to delivering quality work. 

With these changes, your project meets the required standards. Keep up this momentum in your coding journey!

Should you have any further questions, concerns, or need clarifications, don't hesitate to reach out. Remember, continuous improvement is key!

Warm regards and congratulations on this milestone! 🎉🚀
 
## [Optional] suggestions:
- For future pull requests, please notice that your branch name `structure-and-routes` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-structure-and-routes`. This makes it easier to track the changes and the purpose of each branch. 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**